.gitignore

```diff
@@ -11,7 +11,6 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
```

Comment:
Found we were excluding all the `gradle-wrapper.jar` files. Oops. Already excluding build directories so that's good enough.